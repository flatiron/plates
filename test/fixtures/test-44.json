[
  {
    "id": "item-1",
    "mung": "face"
  },
  {}
]

