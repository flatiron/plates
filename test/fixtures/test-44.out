<div data-bind="key">Should not be changed</div>